### PR TITLE
Config for docker authentication in UCR

### PR DESCRIPTION
**Pull request by sascala**
## Description
Two people in two days asked about this: https://groups.google.com/a/dcos.io/forum/?utm_medium=email&utm_source=footer#!msg/users/VKvMkKlZGtM/I4npJop0AQAJ

## Urgency
- [ ] Blocker 
- [X] High
- [ ] Medium

## Requirements
- Test all commands and procedures.
- Build content [locally](https://github.com/dcos/dcos-docs#test-local) and test for formatting/links.
- Add redirects to [dcos-website/redirect-files](https://github.com/dcos/dcos-website#managing-redirects).
- Change all affected versions (e.g. 1.7, 1.8, and 1.9).
- See the [contribution guidelines](https://github.com/dcos/dcos-docs#contributing).

### CODE REVIEW

File: 1.9/deploying-services/containerizers/ucr.md [hunk 1]
```diff
@@ -20,9 +20,15 @@ The Universal Container Runtime offers the following advantages:
 
 1. In the **Container Image** field, enter your container image.
 
+**Note:** If your service needs to [pull Docker images from a private registry](/docs/1.9/deploying-services/private-docker-registry/), you must specify the [`cluster_docker_credentials_path` in your `config.yaml` file before cluster installation](/docs/1.9/installing/custom/configuration-parameters/#cluster_docker_credentials).
```
Comment (joel-hamill) [inline 1.9/deploying-services/containerizers/ucr.md]: Should this be a prerequisite for this task instead of at the end?

Comment (joel-hamill) [inline 1.9/deploying-services/containerizers/ucr.md]: - Suggest `If your service [pulls Docker...`
- Suggest ```...you must specify the [`cluster_docker_credentials_path` in your `config.yaml`](/docs/1.9/installing/custom/configuration-parameters/#cluster_docker_credentials) file before installing DC/OS.```

File: 1.9/deploying-services/containerizers/ucr.md [hunk 2]
```diff
@@ -20,9 +20,15 @@ The Universal Container Runtime offers the following advantages:
...
+
 # Provision Containers with the Universal Container Runtime from the DC/OS CLI
 
-To provision containers with the UCR from the DC/OS CLI, specify the container type `MESOS` and a the appropriate object in your [Marathon application definition](/docs/1.9/deploying-services/creating-services/). Here, we specify a Docker container with the `docker` object.
+To provision containers with the UCR from the DC/OS CLI, specify the container type `MESOS` and a the appropriate object in your [Marathon application definition](/docs/1.9/deploying-services/creating-services/).
+
+**Note:** If your service needs to [pull Docker images from a private registry](/docs/1.9/deploying-services/private-docker-registry/), you must specify the [`cluster_docker_credentials_path` in your `config.yaml` file before cluster installation](/docs/1.9/installing/custom/configuration-parameters/#cluster_docker_credentials).
```
Comment (joel-hamill) [inline 1.9/deploying-services/containerizers/ucr.md]: same comments as above

**Comment by joel-hamill** [comment]
@sascala ping ^

**Comment by sascala** [comment]
@joel-hamill I rebased this, so there are tons of commits. The relevant ones are c14f645 and 1945d5e. NB: I'm not sure about the maturity of the UCR for 1.10.

**Comment by joel-hamill** [comment]
@sascala are you sure this is okay to merge? did you run the `git rebase` command, or how did you do this? generally when i pull in upstream content, it just gets added in - and does not show up as an actual changed file.

**Pull request closed without merging**